site/content/en/latest/user/global-rate-limit.md

```diff
@@ -818,6 +824,7 @@ kubectl rollout restart deployment envoy-gateway -n envoy-gateway-system
 ```
 
 [Global Rate Limiting]: https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/other_features/global_rate_limiting
+[Local rate limiting]: https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/other_features/local_rate_limiting
 [BackendTrafficPolicy]: https://gateway.envoyproxy.io/latest/api/extension_types.html#backendtrafficpolicy
```

Comment:
this is a 404 page, please change it to relative path: `../../api/extension_types#backendtrafficpolicy`